helper_test.go

```diff
@@ -225,7 +225,8 @@ func execute(t *testing.T, expectedExitCode ExitCode) {
 	if err != nil {
 		t.Fatalf("Test setup failure - could not write to tasks-resolved-count.txt file: %v", err)
 	}
-	exitCode := RunWorker()
+	interruptChan := make(chan os.Signal, 1)
+	exitCode := RunWorker(interruptChan)
```

Comment:
Does this allow us to `Ctrl-C` out of tests now?

Reply:
I believe it does, yes. Were we not able to before?

I changed this to match the signature of the function, which I had changed before running the tests (yikes!) :P

Reply:
I believe it does, yes. Certain tests, some of the time. Were we not able to before?

I changed this to match the signature of the function, which I had changed before running the tests (yikes!) :P